scripts/tasks.sh

```diff
@@ -378,4 +381,65 @@ function set_kubeconfig_server() {
     fi
 }
 
+function taint_primaries() {
+    export KUBECONFIG=/etc/kubernetes/admin.conf
+
+    # Rook tolerations
+    if kubectl get namespace rook-ceph &>/dev/null; then
+        kubectl -n rook-ceph patch cephclusters rook-ceph --type=merge -p '{"spec":{"placement":{"all":{"tolerations":[{"key":"node-role.kubernetes.io/master","operator":"Exists"}]}}}}'
```

Comment:
it would be great if there was some way to add an addon hook for this but not gonna make it a blocker.